security/pkg/nodeagent/cache/secretcache.go

```diff
@@ -226,6 +231,26 @@ func (sc *SecretCache) GenerateSecret(ctx context.Context, connectionID, resourc
 	return ns, nil
 }
 
+// SecretExist checks if secret already existed.
+// This API is used for sds server to check if coming request is ack request.
+func (sc *SecretCache) SecretExist(connectionID, resourceName, token, version string) bool {
+	key := ConnKey{
+		ConnectionID: connectionID,
+		ResourceName: resourceName,
+	}
+	val, exist := sc.secrets.Load(key)
+	if !exist {
+		return false
+	}
+
+	e := val.(model.SecretItem)
+	if e.ResourceName == resourceName && e.Token == token && e.Version == version {
```

Comment:
You don't need this if block, just 

return e.ResourceName == resourceName && e.Token == token && e.Version == version

Reply:
done